invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -104,7 +104,7 @@
 
     <!-- PREVIEW HEADER -->
     {% if is_preview %}
-      <div class="ui info flashed bottom attached manage message">
+      <div class="ui info flashed bottom attached manage message record-banner">
```

Comment:
since you moved the css to invenio theme, you should also propagate the template changes to the invenio-theme templates, otherwise it will be fixed only here in the record details template of app rdm